var/spack/repos/builtin/packages/hybpiper/package.py

```diff
@@ -0,0 +1,61 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+import glob
+import os
+
+
+class Hybpiper(Package):
+    """HybPiper was designed for targeted sequence capture, in which DNA
+       sequencing libraries are enriched for gene regions of interest,
+       especially for phylogenetics. HybPiper is a suite of Python scripts
+       that wrap and connect bioinformatics tools in order to extract target
+       sequences from high-throughput DNA sequencing reads"""
+
+    homepage = "https://github.com/mossmatters/HybPiper"
+    url      = "https://github.com/mossmatters/HybPiper/archive/v1.2.0.tar.gz"
+
+    version('1.2.0', '0ad78e9ca5e3f23ae0eb6236b07e1780')
+
+    # Only needed for building docs
+    depends_on('python@2.7:', type='build')
```

Comment:
You say it's only needed for docs, but is it not also needed for `py-biopython`?